src/locutil.rs

```diff
@@ -0,0 +1,39 @@
+// TODO import lineBreakG fn from whitespace
+// TODO define types for all fn parameters and structs
+// TODO add getLineInfo fn
+
+// These are used when `options.locations` is on, for the
+// `startLoc` and `endLoc` properties.
+
+mod locutil {
+    pub struct Position {
+        line: u64,
```

Comment:
would we ever realistically need numbers this big?

Reply:
No assumptions! Column won't be but what can we say about lines?